v2/apiserver/internal/core/mongodb/events_store.go

```diff
@@ -184,6 +189,9 @@ func (e *eventsStore) Cancel(ctx context.Context, id string) error {
 		return errors.Wrapf(err, "error updating status of event %q worker", id)
 	}
 
+	// TODO: If the event was running when it was canceled, its jobs need to be
```

Comment:
Do we need to create a ticket to track this TODO (and the add'l one further down)?

Reply:
I opened https://github.com/brigadecore/brigade/issues/1191